docs/master/subscriptions/client-implementations.md

```diff
@@ -91,13 +91,7 @@ class PusherLink extends ApolloLink {
   }
 
   getChannelFromResponse(response, operation) {
-    return !!response.extensions &&
-      !!response.extensions.lighthouse_subscriptions &&
-      !!response.extensions.lighthouse_subscriptions.channels
-      ? response.extensions.lighthouse_subscriptions.channels[
-          operation.operationName
-        ]
-      : null;
+    return response?.extensions?.lighthouse_subscriptions.channel ?? null;
```

Comment:
Just in case this is added and subscriptions is not (yet) enabled on the server.

```suggestion
    return response?.extensions?.lighthouse_subscriptions?.channel ?? null;
```

Reply:
If `lighthouse_subscriptions` is present, the `channel? key must be present. We don't have to be scared of `null` everywhere ;)